specs/booking.yml

```diff
@@ -440,6 +440,30 @@ paths:
                 Taxi:
                   summary: Taxi
                   externalValue: ../examples/taxi/booking-create-response.json
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/core/error.json
+        '401':
+          description: Authorization error (invalid API key)
+          content:
+            application/json:
+              schema:
+                $ref: ../schemas/core/error.json
+        '409':
+          description: Booking exists for this User. If TSP does not support more than one booking per user, it can indicate it by sending this response when WhimApp creates a new booking.
```

Comment:
i'd add `Booking ID existed` for clarification

Reply:
@hieuunguyeen  Something like this? https://github.com/maasglobal/maas-tsp-api/pull/49/files#diff-240da04d211b876de601db896d2cf21bR268